src/page/OrderResume.page.ts

```diff
@@ -0,0 +1,11 @@
+import { $, ElementFinder, promise } from 'protractor';
+
+export class OrderResumePage {
+  private get payoutResultTitle(): ElementFinder {
```

Comment:
add Label suffix in order to know what locator type it is
`payoutResultTitleLabel`

Reply:
done